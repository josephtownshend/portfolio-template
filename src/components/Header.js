import React from 'react';
import '../App.css';

export default class Header extends React.Component {

  render() {
    return (
      <React.Fragment>
        <div className="NavBar">
          <button onClick={this.props.handleChangeTitle} className="Button_title">Artist Website</button>
          <button onClick={this.props.handleChangeAbout} className="Button_subMenu">About</button>
          <button onClick={this.props.handleChangeWriting} className="Button_subMenu">Writing</button>
        </div>
      </React.Fragment>
    );
  }
}
