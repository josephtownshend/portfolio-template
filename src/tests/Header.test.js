import React from 'react';
import { configure } from 'enzyme';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

configure({ adapter: new Adapter() });

it('shallow renders without crashing', () => {
  shallow(<Header />)
});

it('renders correctly', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
})
