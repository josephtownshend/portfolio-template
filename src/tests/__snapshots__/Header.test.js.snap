// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="NavBar"
>
  <button
    className="Button_title"
  >
    Artist Website
  </button>
  <button
    className="Button_subMenu"
  >
    Contact
  </button>
  <button
    className="Button_subMenu"
  >
    About
  </button>
  <button
    className="Button_subMenu"
  >
    Writing
  </button>
</div>
`;
